.github/workflows/test.yaml

```diff
@@ -286,7 +286,9 @@ jobs:
               echo "::set-output name=already-ran::true"
           fi
     - name: Run spread tests
-      if: "contains(github.event.pull_request.labels.*.name, 'Run nested') && steps.cached-results.outputs.already-ran != 'true'"
+      # run if the commit is pushed to the release/* branch or there is a 'Run
+      # nested' label set on the PR
+      if: "(contains(github.event.pull_request.labels.*.name, 'Run nested') || contains(github.ref, 'refs/heads/release/')) && steps.cached-results.outputs.already-ran != 'true'"
```

Comment:
Having this is great. But I think the current spread has some issue when it tries to run under release/**, e.g.:
https://github.com/snapcore/snapd/runs/1224993441

Reply:
the run you linked to failed on unit tests, not on running spread actually, did you mean a different run?